bower.json

```diff
@@ -24,7 +24,7 @@
     },
     "lightgallery": {
       "main": [
-        "./dist/js/lightgallery.min.js",
+        "./dist/js/lightgallery.js",
```

Comment:
uglify-js messes up with the minified version of lightgallery as can be read here [Uncaught TypeError: i.data is not a function](https://github.com/sachinchoolur/lightGallery/issues/458).
This may be also be fixed with an lightgallery update to 1.5.0.